### ISSUE TITLE
Access denied

### ISSUE THREAD

**Issue by Nick-Elliott-UK** [astrada/google-drive-ocamlfuse#368]
Hi,

I've just upgraded my OS (from Kubuntu 17.04 to 17.10) and no Google Drive is giving permission errors.

I can copy files to and from Google Drive but if I try to delete, rename or move a file I get an "access denied" error.

It doesn't make any difference if Ioad google-drive-ocamlfuse manually or using a startup script.

So I tried using fstab to mount the drive but this gives the error "gdfuse not found".

Perhaps the problems are related, do you have any suggestions?

Thanks

Nick

**Comment by astrada** [comment]
Are the userid/groupid of your user changed after the upgrade?

**Comment by Nick-Elliott-UK** [comment]
No my uid and gid are still 1000

**Comment by astrada** [comment]
Have you already tried restarting from scratch? (Removing `~/.gdfuse/default` and reauthorizing the app?)

**Comment by Nick-Elliott-UK** [comment]
Yes. I created a new profile.

OK you will be happy to hear the problem is definitely to do with permissions on my system and not google-drive-ocamlfuse. I tried to do the same operations on some files in another directory on Google Drive and there was no error.

I've just done a detailed check on all the files and directories. For some reason all the files in just one directory are assigned to "nogroup". This is why I'm getting the error. 

Now I need to work out why or how this happened in the first place.

Thank you for your help, the userid/groupid comment was very useful. Sorry for wasting your time!

Nick
